docs/docs/advanced/useComposedEventHandler.mdx

```diff
@@ -0,0 +1,82 @@
+---
+sidebar_position: 6
+---
+
+# useComposedEventHandler
+
+This is a hook that lets you compose [useEvent](/docs/advanced/useEvent)-based event handlers (such as [useAnimatedScrollHandler](/docs/scroll/useAnimatedScrollHandler) or your own custom ones) into a single, combined event handler.
+
+### Arguments
+
+#### `handlers`
+
+An array of event handlers created using [useEvent](/docs/advanced/useEvent) hook. `useComposedEventHandler` hook reacts to any changes in given handlers and rebuilds whenever it is needed.
+
+### Returns
+
+The hook returns a handler object that can be hooked into any [`Animated component`](/docs/fundamentals/glossary#animated-component).
+The handler should be passed to any `onEvent` (e.g. `onScroll`) parameter regardless of what events it handles.
```

Comment:
This sentence is confusing. I think we should encourage more intuitive usage here, passing the handler to an event handler property that is related to the actual composed handler, e.g. merged `onScroll` handlers should go to `onScroll` property.